appintro/src/main/res/layout/appintro_intro_layout2.xml

```diff
@@ -32,9 +32,11 @@
     <ImageButton
         android:id="@+id/skip"
         style="@style/AppIntro2ButtonStyleCompat"
+        android:layout_height="0dp"

```

Comment:
Please remove this as it causes the button to don't be a perfect square anymore.